### PR TITLE
Wording proposal by Gorry on exposing numbers of streams and such

### PR DESCRIPTION
**Pull request by mirjak**

### CODE REVIEW

**Comment by mirjak** [comment]
The proposed wording makes sentence stronger, more list a SHOULD requirement. Therefore we wanted to check with others before we merge.

File: draft-ietf-quic-applicability.md [hunk 1]
```diff
@@ -327,9 +327,9 @@ for that message.
 If a QUIC receiver has opened the maximum allowed concurrent
 streams, and the sender indicates that more streams are needed, it
 doesn't automatically lead to an increase of the maximum number of streams by
-the receiver. Therefore it can be valuable to expose the maximum number of
-allowed, currently open, and currently used streams to the application to make
-the mapping of data to streams dependent on this information.
+the receiver. Therefore, an application can utilise the maximum number of
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:330]: S/utilise/use

**Review by martinthomson — APPROVED** [review]
Seems simpler and clearer. I don't see this as normative.

File: draft-ietf-quic-applicability.md (hunk 1 above)
Comment (britram) [inline draft-ietf-quic-applicability.md:330]:
```suggestion
the receiver. Therefore, an application can use the maximum number of
```

**Pull request merged**